plasmapy/formulary/ionization.py

```diff
@@ -88,3 +88,84 @@ def ionization_balance(n: u.m ** -3, T_e: u.K) -> u.dimensionless_unscaled:
 
 Z_bal = ionization_balance
 """alias for :func:`ionization_balance`"""
+
+
+@validate_quantities(
+    T_e={"can_be_negative": False, "equivalencies": u.temperature_energy()}
+)
+def Saha(g_j, g_k, n_e: u.m ** -3, E_jk: u.J, T_e: u.K) -> u.dimensionless_unscaled:
```

Comment:
Does everyone like `g_j`, `g_k` for these arg names? If it's conventional enough I'm fine with it